.circleci/config.yml

```diff
@@ -465,6 +465,9 @@ jobs:
           elif [[ ${BUILD_ENVIRONMENT} == *"parallelnative"* ]]; then
             echo 'ATEN_THREADING=NATIVE' >> "${BASH_ENV}"
           fi
+          if [[ ${BUILD_ENVIRONMENT} == *"linux_xenial_cuda"* ]]; then
+            echo 'USE_FAST_NVCC=ON' >> "${BASH_ENV}"
+          fi
```

Comment:
I don't think we need to enable it for CI testing, as the code is already compiled for just one architecture